### PR TITLE
fix(plasma-ui): hide controls when disabled or isSingleItem state

### PR DESCRIPTION
**Pull request by Yakutoc**
Скрываем стрелочки если какой-либо барабан имеет **только** одно значение или у него выключенное (disabled) состояние. 

:baby_chick: Download canary assets:

[colors--canary.245.3564432469.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/colors--canary.245.3564432469.xml)  
[color_sbermarket_ios-swift--canary.245.3564432469.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_ios-swift--canary.245.3564432469.swift)  
[color_sbermarket_kotlin--canary.245.3564432469.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_kotlin--canary.245.3564432469.kt)  
[color_sbermarket_react-native--canary.245.3564432469.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_react-native--canary.245.3564432469.ts)  
[color_sberprime_ios-swift--canary.245.3564432469.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_ios-swift--canary.245.3564432469.swift)  
[color_sberprime_kotlin--canary.245.3564432469.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_kotlin--canary.245.3564432469.kt)  
[color_sberprime_react-native--canary.245.3564432469.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_react-native--canary.245.3564432469.ts)  
[PlasmaTokensColor--canary.245.3564432469.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/PlasmaTokensColor--canary.245.3564432469.swift)  
[typo_mage_kotlin--canary.245.3564432469.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_kotlin--canary.245.3564432469.kt)  
[typo_mage_react-native--canary.245.3564432469.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_react-native--canary.245.3564432469.ts)  
[typo_plasma_kotlin--canary.245.3564432469.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_kotlin--canary.245.3564432469.kt)  
[typo_plasma_react-native--canary.245.3564432469.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_react-native--canary.245.3564432469.ts)  
[typo_ruler_kotlin--canary.245.3564432469.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_kotlin--canary.245.3564432469.kt)  
[typo_ruler_react-native--canary.245.3564432469.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_react-native--canary.245.3564432469.ts)  
[typo_sage_kotlin--canary.245.3564432469.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_kotlin--canary.245.3564432469.kt)  
[typo_sage_react-native--canary.245.3564432469.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_react-native--canary.245.3564432469.ts)  
[typo_sbermarket_kotlin--canary.245.3564432469.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_kotlin--canary.245.3564432469.kt)  
[typo_sbermarket_react-native--canary.245.3564432469.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_react-native--canary.245.3564432469.ts)  
[typo_soulmate_kotlin--canary.245.3564432469.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_kotlin--canary.245.3564432469.kt)  
[typo_soulmate_react-native--canary.245.3564432469.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_react-native--canary.245.3564432469.ts)

<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @salutejs/plasma-temple@1.114.4-canary.245.3564432469.0
  npm install @salutejs/plasma-ui@1.147.3-canary.245.3564432469.0
  # or 
  yarn add @salutejs/plasma-temple@1.114.4-canary.245.3564432469.0
  yarn add @salutejs/plasma-ui@1.147.3-canary.245.3564432469.0
  ```
</details>

### CODE REVIEW

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/demo-canvas-app-pr-245/
  <details>
    <summary>New documentation</summary>
    <br />

    http://plasma.fine.sberdevices.ru/pr/demo-canvas-app-pr-245/

  </details>

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr-245/
  ui storybook: http://plasma.sberdevices.ru/pr-245/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr-245/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr-245/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr-245/temple-storybook/
  <details>
    <summary>New documentation</summary>
    <br />

    website: http://plasma.fine.sberdevices.ru/pr/pr-245/
    ui storybook: http://plasma.fine.sberdevices.ru/pr/pr-245/ui-storybook/
    web storybook: http://plasma.fine.sberdevices.ru/pr/pr-245/web-storybook/
    b2c storybook: http://plasma.fine.sberdevices.ru/pr/pr-245/b2c-storybook/
    temple storybook: http://plasma.fine.sberdevices.ru/pr/pr-245/temple-storybook/

  </details>

File: packages/plasma-ui/src/components/Pickers/Picker.tsx [hunk 1]
```diff
@@ -484,6 +484,8 @@ export const Picker = ({
         [virtualItems, infiniteScroll, max, min, index, isSingleItem],
     );
 
+    const hasControls = controls && !disabled;
```
Comment (Yeti-or) [inline packages/plasma-ui/src/components/Pickers/Picker.tsx:487]: а как это работает ? 
если есть контролы и не выключен

каким образом мы ориентируемся на кол-во items?

Comment (Yakutoc) [reply packages/plasma-ui/src/components/Pickers/Picker.tsx:487]: @Yeti-or Привет 

> каким образом мы ориентируемся на кол-во items?

На 296 строчке у нас вычисляется состояние для **disabled**

```const disabled = rest.disabled || isSingleItem;```

Comment (Yakutoc) [reply packages/plasma-ui/src/components/Pickers/Picker.tsx:487]: @Yeti-or Единственное что поправлю это пункт про 

> readOnly - фокус возможен

Comment (Yakutoc) [reply packages/plasma-ui/src/components/Pickers/Picker.tsx:488]: > @Yeti-or Единственное что поправлю это пункт про
> 
> > readOnly - фокус возможен

Решено вынести в отдельную [задачу](https://github.com/salute-developers/plasma/issues/247)

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/demo-canvas-app-pr-245/
  <details>
    <summary>New documentation</summary>
    <br />

    http://plasma.fine.sberdevices.ru/pr/demo-canvas-app-pr-245/

  </details>

**Review by neretin-trike — APPROVED** [review]

**Review by Yeti-or — APPROVED** [review]

**Comment by Yeti-or** [comment]
@Yakutoc сделай мне пожалуйста два скриншота, руками: Как было и как стало
чтобы в блог вставить

**Pull request merged**